CMakeLists.txt

```diff
@@ -36,6 +36,12 @@ add_custom_target(luatest
   COMMENT "Run regression tests"
 )
 
+add_custom_target(luatest-no-coverage
+  COMMAND ${LUATEST} -v --shuffle all:${seed}
+  WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
```

Comment:
In the target above, `CMAKE_CURRENT_SOURCE_DIR` is used. Let's do the same.

Reply:
Fixed.